### DISCUSSION TITLE
Verrukt in "reflux edition", is it ever coming to actual game?

### DISCUSSION THREAD

**Discussion by HalfBlu3** [nzp-team/nzportable#525] [General]
I found the old leaked version of the game, reflux edition, and I was curious what maps it had. I saw that the game had Verrukt, so I tried putting it into my current 64-bit installation. It sorta worked, but zombie pathing was broken, not all collision worked, wallbuys were missing etc, and I know that it's because it was just the `.bsp` file, and it needs a `.way`,`.way`,`.mbox`, etc. are there any plans to bring this map back at any point? I'd love to have it playable in the game, but don't have the technical knowledge to just fix it myself.

**Comment by MotoLegacy** [comment]
Hi, yes Verruckt is the next major release planned from me, but I am not using the old nasty leaked build from 2011, for a lot of reasons (performance, architecture, scale, pride, etc.).

**Reply by Laser327** [reply]
I have a question about Jugger-Nog being present in Verrückt.
Once Verrückt is finished.

Is Jugger-Nog gonna function the same as in Verrückt and Shi No Numa (Reduce health regeneration delay and increase players health to 160 instead of 250) or its just gonna function the same as the one in Der Riese?


Heres a vid on how Jug works on Verrückt and how it works in Shi No Numa's Dog rounds (might be useful for later, or not idk)
https://www.youtube.com/watch?v=zBiqfXZL8Uw

**Reply by MotoLegacy** [reply]
Hi, thanks for the question. Jugger-Nog is going to function the same as it does currently, that is the "post-Shi No Numa" implementation. I'm very selective with how I pull specific features, targeting _World at War_ but with the refinements iterated upon in _Black Ops_, Jugger-Nog behaving consistently across the entire game being one of those things.

That being said, I'll probably opt for Double Tap v1 now that I've implemented that, v2 for official maps feels a little too intrusive.

**Reply by Laser327** [reply]
With how slowly the zombies attack in NZ:P (especially with the absence of double swiping) and how broken the regenration delay of "Pre-Der Riese" Jug was, it just makes much more sense to keep Jug as is.

im not gonna ask a question about Verrückt sprinters since Crusader told me that sprinters are gonna be incuded.
But either way, thanks for letting us know.

**Reply by MotoLegacy** [reply]
Hmmm... double swiping is absent. Thanks for the reminder. Will get this added soon.

**Reply by BCDeshiG** [reply]
Forgive my ignorance but wasn't double swiping a bo2 feature?

**Reply by MotoLegacy** [reply]
IIRC it had a higher rate and was generally faster in Black Ops II, but the mechanic itself has always been there(? at least since Black Ops)

**Reply by Laser327** [reply]
From what i remember, Double Swiping was apparently introduced with Verrückt despite the fact that ive played a little bit of World at War's Nacht Der Untoten and double swiping was present there already.

Its just that the AI in WaW's Nacht is pretty slow and also lack that annoying grabbing mechanic that was present with the introduction of Verrückt.

**Reply by MotoLegacy** [reply]
Ding ding ding — and it's in game now, along with generally faster attack opportunities. Enjoy the difficulty bump!
